### PR TITLE
conversation

### PR DESCRIPTION
**Pull request by wenjinglee1104**

### CODE REVIEW

**Comment by lamberta** [comment]
hank you, @wenjinglee1104 .
Please sign the CLA: https://cla.developers.google.com/ and we can merge after review.
After signing, you'll need to add a comment to this pull request that says: `I signed it!`

**Comment by wenjinglee1104** [comment]
I signed it!

File: site/zh-cn/tutorials/keras/conversion.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+**Keras model**: Keras 模型通常保存为 HDF5 文件。有关保存 Keras 模型的更多信息，请访问 [此处](https://keras.io/getting-started/faq/#savingloading-whole-models-architecture-weights-optimizer-state)。
+
+**TensorFlow Hub module**: 可在 TensorFlow Hub 上进行分发的已打包模型，TensorFlow Hub 是一个共享和发现模型的平台。模型库见 [此处](tfhub.dev)。
+
+根据您尝试转换的模型类型, 您需要将不同的参数传递给转换器。比如, 假设您保存了一个名为 `model.h5` 的 Keras 模型到 `tmp/` 目录。为了使用 TensorFlow.js 转换器转换模型, 您可以运行以下命令:
+
+    $ tensorflowjs_converter --input_format=keras /tmp/model.h5 /tmp/tfjs_model
+
+这将会在 `/tmp/model.h5` 转换模型并且输出 `model.json` 文件和二进制权重文件到目录 `tmp/tfjs_model/` 。
```
Comment (kuri-leo) [inline site/zh-cn/tutorials/keras/conversion.md]:
```suggestion
这会将路径为 `/tmp/model.h5` 的模型转换并且输出 `model.json` 文件及其二进制权重文件到目录 `tmp/tfjs_model/` 中。
```

File: site/zh-cn/tutorials/keras/conversion.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+有关不同模型格式对应的命令行参数的更多信息，请参阅 tensorflow.js 转换器 [自述文件](https://github.com/tensorflow/tfjs-converter)。
+
+在转换过程中，我们会遍历模型图并确认 TensorFlow.js 是否支持每个操作。如果是支持的，我们将图编写成浏览器可以使用的格式。我们尝试通过将权重分成4MB的文件来优化模型以便在 Web 上使用 - 这样它们就可以被浏览器缓存。我们也尝试使用开源 [Grappler](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/core/grappler) 简化模型图。图形的简化包括相邻操作的折叠，消除常见子图像等。这些更改对模型的输出没有影响。为了进一步优化，用户可以输入参数指示转换器将模型量化到特定的字节大小。量化是一种通过用更少的比特表示权重以此减少模型大小的技术。用户应务必确保量化后模型的准确度保持在可接受范围。
```
Comment (kuri-leo) [inline site/zh-cn/tutorials/keras/conversion.md]:
```suggestion
在转换过程中，我们会遍历模型图形并确认 TensorFlow.js 是否支持每个操作。如果是支持的，我们将图转换成浏览器可以使用的格式。我们尝试通过将权重分成4MB的文件来优化模型以便在 Web 上使用 - 这样它们就可以被浏览器缓存。我们也尝试使用开源的 [Grappler](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/core/grappler) 来简化模型图形。图形的简化包括相邻操作的折叠，消除常见子图像等。这些更改对模型的输出没有影响。为了进一步优化，用户可以输入参数指示转换器将模型量化到特定的字节大小。量化是一种通过用更少的比特表示权重以此减少模型大小的技术。用户应务必确保量化后模型的准确度保持在可接受范围。
```

File: site/zh-cn/tutorials/keras/conversion.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+如果在转换过程中遇到了不支持的操作，则该过程失败，我们将为用户打印出该操作的名称。请将此提交到我们的 [GitHub](https://github.com/tensorflow/tfjs/issues)  - 我们会尝试根据用户的需求实施新的操作。
```
Comment (kuri-leo) [inline site/zh-cn/tutorials/keras/conversion.md]:
```suggestion
如果在转换过程中遇到了不支持的操作，则该过程失败，我们将为用户打印出该操作的名称。请将此提交到我们的 [GitHub](https://github.com/tensorflow/tfjs/issues)  - 我们会尝试根据用户的需求实现更多新的操作。
```

File: site/zh-cn/tutorials/keras/conversion.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+### 最佳做法
+
+虽然在转换过程中我们尽力优化您的模型, 但通常确保您的模型顺利运行的最佳方式是考虑资源受限的环境下构建。这意味着避免过度复杂的建构和尽可能减少参数（权重）的数目。
+
+## 运行您的模型
+
+成功转换模型之后，您将得到一组权重文件和一个模型拓扑文件。TensorFlow.js 提供模型加载 APIs ，您可以使用这些接口获取模型并且在浏览器中运行推断。
+
+以下是转换后的 TensorFlow SavedModel 或 TensorFlow Hub模块的 API :
```
Comment (kuri-leo) [inline site/zh-cn/tutorials/keras/conversion.md]:
```suggestion
以下是加载转换后的 TensorFlow SavedModel 或 TensorFlow Hub模块的 API :
```

File: site/zh-cn/tutorials/keras/conversion.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
+# 模型转换
+
+TensorFlow.js 配备了各种预训练模型，这些模型可以浏览器中使用， [模型版本库](https://github.com/tensorflow/tfjs-models) 中有相关介绍。但是，您可能已经在其他地方找到或创建了一个 TensorFlow 模型，希望在 Web 应用程序中使用该模型。为此，TensorFlow.js 提供了一个模型 [转换器](https://github.com/tensorflow/tfjs-converter) 。TensorFlow.js 转换器有两个组件:
```
[the remark this answers is not in the document]
Comment (wenjinglee1104) [reply site/zh-cn/tutorials/keras/conversion.md]: 原文里 a model converter，只有converter是链接欸

File: site/zh-cn/tutorials/keras/conversion.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
+# 模型转换
+
+TensorFlow.js 配备了各种预训练模型，这些模型可以浏览器中使用， [模型版本库](https://github.com/tensorflow/tfjs-models) 中有相关介绍。但是，您可能已经在其他地方找到或创建了一个 TensorFlow 模型，希望在 Web 应用程序中使用该模型。为此，TensorFlow.js 提供了一个模型 [转换器](https://github.com/tensorflow/tfjs-converter) 。TensorFlow.js 转换器有两个组件:
+
+1. 一个命令行程序，用于在 TensorFlow.js 中转换 Keras 和 TensorFlow 模型。
```
[the remark this answers is not in the document]
Comment (wenjinglee1104) [reply site/zh-cn/tutorials/keras/conversion.md]: 嗯，thanks.

File: site/zh-cn/tutorials/keras/conversion.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
+# 模型转换
+
+TensorFlow.js 配备了各种预训练模型，这些模型可以浏览器中使用， [模型版本库](https://github.com/tensorflow/tfjs-models) 中有相关介绍。但是，您可能已经在其他地方找到或创建了一个 TensorFlow 模型，希望在 Web 应用程序中使用该模型。为此，TensorFlow.js 提供了一个模型 [转换器](https://github.com/tensorflow/tfjs-converter) 。TensorFlow.js 转换器有两个组件:
+
+1. 一个命令行程序，用于在 TensorFlow.js 中转换 Keras 和 TensorFlow 模型。
+2. 一个 API，用于在浏览器中使用 TensorFlow.js 加载和执行模型。
+
+## 转换您的模型
+
+TensorFlow.js 转换器使用几种不同的模型格式:
```
[the remark this answers is not in the document]
Comment (wenjinglee1104) [reply site/zh-cn/tutorials/keras/conversion.md]: 嗯，thanks.

File: site/zh-cn/tutorials/keras/conversion.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+TensorFlow.js 配备了各种预训练模型，这些模型可以浏览器中使用， [模型版本库](https://github.com/tensorflow/tfjs-models) 中有相关介绍。但是，您可能已经在其他地方找到或创建了一个 TensorFlow 模型，希望在 Web 应用程序中使用该模型。为此，TensorFlow.js 提供了一个模型 [转换器](https://github.com/tensorflow/tfjs-converter) 。TensorFlow.js 转换器有两个组件:
+
+1. 一个命令行程序，用于在 TensorFlow.js 中转换 Keras 和 TensorFlow 模型。
+2. 一个 API，用于在浏览器中使用 TensorFlow.js 加载和执行模型。
+
+## 转换您的模型
+
+TensorFlow.js 转换器使用几种不同的模型格式:
+
+**SavedModel**: 保存 TensorFlow 模型的默认格式。 SavedModel 的格式  [在此处记录](https://www.tensorflow.org/guide/saved_model)。
```
[the remark this answers is not in the document]
Comment (wenjinglee1104) [reply site/zh-cn/tutorials/keras/conversion.md]: Thanks.

File: site/zh-cn/tutorials/keras/conversion.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+
+**Keras model**: Keras 模型通常保存为 HDF5 文件。有关保存 Keras 模型的更多信息，请访问 [此处](https://keras.io/getting-started/faq/#savingloading-whole-models-architecture-weights-optimizer-state)。
+
+**TensorFlow Hub module**: 可在 TensorFlow Hub 上进行分发的已打包模型，TensorFlow Hub 是一个共享和发现模型的平台。模型库见 [此处](tfhub.dev)。
```
[the remark this answers is not in the document]
Comment (wenjinglee1104) [reply site/zh-cn/tutorials/keras/conversion.md]: 第一个地方我翻译的时候就觉得好拗口，可是当时又没有想出好的说法,thanks.后面一句没有更改吧。

File: site/zh-cn/tutorials/keras/conversion.md (hunk 1 above)
Comment (wenjinglee1104) [reply site/zh-cn/tutorials/keras/conversion.md]: Thanks.

File: site/zh-cn/tutorials/keras/conversion.md (hunk 2 above)
Comment (wenjinglee1104) [reply site/zh-cn/tutorials/keras/conversion.md]: Thanks.

File: site/zh-cn/tutorials/keras/conversion.md (hunk 3 above)
Comment (wenjinglee1104) [reply site/zh-cn/tutorials/keras/conversion.md]: 棒欸，thanks.

File: site/zh-cn/tutorials/keras/conversion.md (hunk 4 above)
Comment (wenjinglee1104) [reply site/zh-cn/tutorials/keras/conversion.md]: 唔唔，学到了，thanks.

**Comment by kuri-leo** [comment]
@wenjinglee1104 姐姐不是让你写thanks啊，是把这个commits融进去啊

**Comment by wenjinglee1104** [comment]
That’s all.

**Pull request closed without merging**